boards/DIM/Inc/FreeRTOSConfig.h

```diff
@@ -174,8 +174,8 @@ standard names. */
 #define xPortSysTickHandler SysTick_Handler
 
 /* USER CODE BEGIN Defines */
-/* Section where parameter definitions can be added (for instance, to override
- * default ones in FreeRTOS.h) */
+// Required for thread-safety of newlib sprintf, strtok, etc...
```

Comment:
```suggestion
// Integrates the Tracealyzer recorder with FreeRTOS. Tracealyzer recommends
// that this is inserted at the end of FreeRTOSConfig.h.
#if (configUSE_TRACE_FACILITY == 1)
#include "trcRecorder.h"
#endif

// Required for thread-safety of newlib sprintf, strtok, etc...
```

Reply:
`#define configUSE_TRACE_FACILITY 1`
Isn't defined in this file either

Reply:
Yeah i'm gonna do that in another PR

Reply:
And fully test the tracing before i make the changes